scripts/Tools/concat_daily_hist.csh

```diff
@@ -1,71 +0,0 @@
-#!/bin/csh -f
```

Comment:
It did, this was the only file for which support was expressed.   I will add it back.